views/js/navigation.js

```diff
@@ -1,10 +1,15 @@
+let pacmanLoadEvent = new Event("pacmanLoad");
+
 const changeActivePage = (url) => {
   changeIframePage(url);
   changeChatRoom(url);
 }
 
 const changeIframePage = (url) => {
   document.getElementById('kubercade_iframe').src = url;
+  if (url == '/games/pacman') {
```

Comment:
```suggestion
  if (url === '/games/pacman') {
```
Almost always use triple equals in JS to avoid unexpected type coercion.

Reply:
Fixed 02ea219